server.go

```diff
@@ -65,8 +65,15 @@ import (
 	stats "github.com/libp2p/go-libp2p-gorpc/stats"
 )
 
+const (
+	// ContextKeyRequestSender is default key for RPC service function context to retrieve peer ID of current request sender
+	ContextKeyRequestSender = "request_sender"
+)
+
 var logger = logging.Logger("p2p-gorpc")
 
+var ErrLocalCall = errors.New("sender is missing: local call")
```

Comment:
I am slightly inclined to let the `Call` method set the local peer ID so that all calls, remote or local, have this set. It seems a more consistent approach given that "local calls" are a transparent shortcut.